### ISSUE TITLE
in_tail: timerfc_settime error

### ISSUE THREAD

**Issue by AkashKrDutta** [fluent/fluent-bit#670]
I have the following error that is present in fluentbit -
```
[2018/07/02 16:45:33] [ info] [engine] started (pid=1)
[2018/07/02 16:45:33] [ info] [out_kafka] brokers='kafka-0.kube4.skae.tower-research.com:9092,kafka-1.kube4.skae.tower-research.com:9092,kafka-2.kube4.skae.tower-research.com:9092' topics='containers-logs-fluentbit.stream'
[2018/07/02 16:45:33] [ info] [out_kafka] brokers='kafka-0.kube4.skae.tower-research.com:9092,kafka-1.kube4.skae.tower-research.com:9092,kafka-2.kube4.skae.tower-research.com:9092' topics='kubelet-logs-fluentbit.stream'
[2018/07/02 16:45:33] [ info] [out_kafka] brokers='kafka-0.kube4.skae.tower-research.com:9092,kafka-1.kube4.skae.tower-research.com:9092,kafka-2.kube4.skae.tower-research.com:9092' topics='docker-logs-fluentbit.stream'
[2018/07/02 16:45:33] [ info] [out_kafka] brokers='kafka-0.kube4.skae.tower-research.com:9092,kafka-1.kube4.skae.tower-research.com:9092,kafka-2.kube4.skae.tower-research.com:9092' topics='systemd-logs-fluentbit.stream'
[2018/07/02 16:45:33] [ info] [filter_kube] https=1 host=kubernetes.default.svc.cluster.local port=443
[2018/07/02 16:45:33] [ info] [filter_kube] local POD info OK
[2018/07/02 16:45:33] [ info] [filter_kube] testing connectivity with API server...
[2018/07/02 16:45:33] [ info] [filter_kube] API server connectivity OK
[2018/07/02 16:45:33] [ info] [http_server] listen iface=0.0.0.0 tcp_port=2020
% Failed to produce to topic systemd-logs-fluentbit.stream: Local: Queue full
[2018/07/02 16:45:35] [ warn] [out_kafka] internal queue is full, retrying in one second
% Failed to produce to topic systemd-logs-fluentbit.stream: Local: Queue full
[2018/07/02 16:45:36] [ warn] [out_kafka] internal queue is full, retrying in one second
% Failed to produce to topic systemd-logs-fluentbit.stream: Local: Queue full
[2018/07/02 16:45:37] [ warn] [out_kafka] internal queue is full, retrying in one second
% Failed to produce to topic systemd-logs-fluentbit.stream: Local: Queue full
[2018/07/02 16:45:38] [ warn] [out_kafka] internal queue is full, retrying in one second
% Failed to produce to topic systemd-logs-fluentbit.stream: Local: Queue full
[2018/07/02 16:45:39] [ warn] [out_kafka] internal queue is full, retrying in one second
% Failed to produce to topic systemd-logs-fluentbit.stream: Local: Queue full
[2018/07/02 16:45:40] [ warn] [out_kafka] internal queue is full, retrying in one second
% Failed to produce to topic systemd-logs-fluentbit.stream: Local: Queue full
[2018/07/02 16:45:41] [ warn] [out_kafka] internal queue is full, retrying in one second
% Failed to produce to topic systemd-logs-fluentbit.stream: Local: Queue full
[2018/07/02 16:45:42] [ warn] [out_kafka] internal queue is full, retrying in one second
% Failed to produce to topic systemd-logs-fluentbit.stream: Local: Queue full
[2018/07/02 16:45:43] [ warn] [out_kafka] internal queue is full, retrying in one second
% Failed to produce to topic systemd-logs-fluentbit.stream: Local: Queue full
[2018/07/02 16:45:44] [ warn] [out_kafka] internal queue is full, retrying in one second
Fluent-Bit v0.13.4
Copyright (C) Treasure Data

[2018/07/02 20:30:08] [  Error] timerfd_settime: Invalid argument, errno=22 at /tmp/fluent-bit-0.13.4/lib/monkey/mk_core/mk_event_epoll.c:189
[2018/07/02 20:30:08] [ warn] [sched] retry for task 56 could not be scheduled
[2018/07/02 20:36:33] [ warn] [sched] retry for task 509 could not be scheduled
[2018/07/02 20:36:33] [  Error] timerfd_settime: Invalid argument, errno=22 at /tmp/fluent-bit-0.13.4/lib/monkey/mk_core/mk_event_epoll.c:189
[2018/07/02 20:51:23] [  Error] timerfd_settime: Invalid argument, errno=22 at /tmp/fluent-bit-0.13.4/lib/monkey/mk_core/mk_event_epoll.c:189
[2018/07/02 20:51:23] [ warn] [sched] retry for task 151 could not be scheduled

```
The last lines of error are repeated many times.

The configuration is - 
```
input-systemd.conf: |
    [INPUT]
      Name              systemd
      Tag               cdp.systemd
      Path              /var/log/journal
      Systemd_Filter    --unit=systemd*
      Parser            systemd

input-docker.conf: |
    [INPUT]
      Name              systemd
      Tag               cdp.docker
      Path              /var/log/journal
      Systemd_Filter    _SYSTEMD_UNIT=docker.service
      Parser            systemd
      # this reads the latest journald logs when restarted
      Read_From_Tail    true

  input-kubelet.conf: |
    [INPUT]
      Name              systemd
      Tag               cdp.kubelet
      Path              /var/log/journal
      Systemd_Filter    _SYSTEMD_UNIT=kubelet.service
      Parser            systemd
      # this reads the latest journald logs when restarted
      Read_From_Tail    true

input-container.conf: |
   [INPUT]
       Name              tail
       Tag               kube.*
       Path              /var/log/containers/*.log
       Parser            docker
       Mem_Buf_Limit     5MB
       Refresh_Interval  5
       Buffer_Chunk_Size 128k
       Buffer_Max_Size   4096k



filter-docker.conf: |
    [FILTER]
      Name record_modifier
      Match cdp.docker
      Record tag cdp.docker

  filter-kubelet.conf: |
    [FILTER]
      Name record_modifier
      Match cdp.kubelet
      Record tag cdp.kubelet

  filter-systemd.conf: |
    [FILTER]
      Name record_modifier
      Match cdp.systemd
      Record tag cdp.systemd
  
 filter-containers.conf: |

    [FILTER]
      Name                kubernetes
      Match               kube.*
      Kube_URL            https://kubernetes.default.svc.cluster.local:443
      Merge_Log           On
      K8S-Logging.Parser  On
      # Use_Journal         On

    [FILTER]
      Name        record_modifier
      Match       kube.*
      Record      tag cdp.containers

   output-containers.conf: |
   [OUTPUT]
       Name           kafka
       Match          kube.*
       Brokers        kafka-0:9092
       Topics         containers-logs-fluentbit.stream
       Timestamp_Key  @timestamp
       Retry_Limit    false
       # hides errors "Receive failed: Disconnected" when kafka kills idle connections
       rdkafka.log.connection.close false
       # producer buffer is not included in http://fluentbit.io/documentation/0.12/configuration/memory_usage.html#estimating
       rdkafka.queue.buffering.max.kbytes 10240
       # for logs you'll probably want this ot be 0 or 1, not more
       rdkafka.request.required.acks 1

    output-systemd.conf: |
    [OUTPUT]
        Name           kafka
        Match          cdp.systemd
        Brokers        kafka-0:9092
        Topics         systemd-logs-fluentbit.stream
        Timestamp_Key  @timestamp
        Retry_Limit    false
        # hides errors "Receive failed: Disconnected" when kafka kills idle connections
        rdkafka.log.connection.close false
        # producer buffer is not included in http://fluentbit.io/documentation/0.12/configuration/memory_usage.html#estimating
        rdkafka.queue.buffering.max.kbytes 10240
        # for logs you'll probably want this ot be 0 or 1, not more
        rdkafka.request.required.acks 1


  output-docker.conf: |
    [OUTPUT]
        Name           kafka
        Match          cdp.docker
        Brokers        kafka-0:9092
        Topics         docker-logs-fluentbit.stream
        Timestamp_Key  @timestamp
        Retry_Limit    false
        # hides errors "Receive failed: Disconnected" when kafka kills idle connections
        rdkafka.log.connection.close false
        # producer buffer is not included in http://fluentbit.io/documentation/0.12/configuration/memory_usage.html#estimating
        rdkafka.queue.buffering.max.kbytes 10240
        # for logs you'll probably want this ot be 0 or 1, not more
        rdkafka.request.required.acks 1

  output-kubelet.conf: |
    [OUTPUT]
        Name           kafka
        Match          cdp.kubelet
        Brokers        kafka-0:9092
        Topics         kubelet-logs-fluentbit.stream
        Timestamp_Key  @timestamp
        Retry_Limit    false
        # hides errors "Receive failed: Disconnected" when kafka kills idle connections
        rdkafka.log.connection.close false
        # producer buffer is not included in http://fluentbit.io/documentation/0.12/configuration/memory_usage.html#estimating
        rdkafka.queue.buffering.max.kbytes 10240
        # for logs you'll probably want this ot be 0 or 1, not more
        rdkafka.request.required.acks 1



```

I don't know how to deal with this error. Surely, I am able to see missing logs in the kafka output topics.
Any help is appreciated. Thanks.

**Comment by judexzhu** [comment]
I got the same issue.

fixed by comment 

```
       # hides errors "Receive failed: Disconnected" when kafka kills idle connections
       rdkafka.log.connection.close false
       # producer buffer is not included in http://fluentbit.io/documentation/0.12/configuration/memory_usage.html#estimating
       rdkafka.queue.buffering.max.kbytes 10240
       # for logs you'll probably want this ot be 0 or 1, not more
       rdkafka.request.required.acks 1
```

**Comment by edsiper** [comment]
Fixed in v1.1.1 release:

https://fluentbit.io/announcements/v1.1.1/
